pymess/config.py

```diff
@@ -166,14 +172,19 @@ def get_email_template_model():
     return get_model(settings.EMAIL_TEMPLATE_MODEL)
 
 
-def get_sms_sender():
+def get_sms_sender(recipient=None, is_testing=None):
     """
     Function returns SMS sender backend from string defined in Pymess settings
     """
-    return import_string(settings.SMS_SENDER_BACKEND)()
+    testing_phones = settings.SMS_TESTING_PHONES
+    if is_testing == True or is_testing is None and (testing_phones and recipient in testing_phones):
```

Comment:
the is_testing is weird this shoud be in the configuration only

Reply:
removed